app/snippets/content_templates/isaacQuiz.json

```diff
@@ -4,6 +4,17 @@
     "title": "New Quiz",
     "visibleToStudents": false,
     "published": false,
+    "rubric": {
+      "type": "content",
+      "encoding": "markdown",
+      "children": [
+        {
+          "type": "content",
+          "value": "_Enter rubric here or remove this section if not wanted_",
```

Comment:
Is it actually possible to remove the section? If so, this wording is fine, and ignore me!

Reply:
You can press the cross in the visual editor to remove rubric and prevent it appearing on a quiz page, do you think some other wording would be better?

Reply:
If the cross leave `rubric` null, or out of the JSON entirely, then nope this is great!